tests/baselines/reference/genericIsNeverEmptyObject.types

```diff
@@ -2,18 +2,18 @@
 // Repro from #29067
 
 function test<T extends { a: string }>(obj: T) {
->test : <T extends { a: string; }>(obj: T) => Pick<T, Exclude<keyof T, "a">> & { b: string; }
+>test : <T extends { a: string; }>(obj: T) => Omit<T, "a", Exclude<keyof T, "a">> & { b: string; }

```

Comment:
I think the self-generating third argument is at least as bad as converting Omit to its definition.

Reply:
Do you think it's worth investing in hiding that argument when it matches the default? (As it always should)

Reply:
Well, it solves one problem -- reading the type -- but not the second problem: writing the type.

And if it applies everywhere, the mismatch might just be confusing. Actually, hiding any optional item entirely is likely to be confusing.

Reply:
I think I’d prefer <https://github.com/microsoft/TypeScript/pull/31205#discussion_r280879188>:
> ```ts
> type Omit<T, K extends keyof any> = T extends unknown ? Pick<T, Exclude<keyof T, K>> : never;
> ```